### ISSUE TITLE
Hard code accessibility strings in S2015R release branch

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/forces-and-motion-basics#171]
For accessibility, many (very long) strings have been added to the strings.json file that should not be provided to translators yet.  We should make sure that these strings do not show up on the translation utility or remove them before the sim is redeployed.

**Comment by jessegreenberg** [comment]
Taking a look at Rosetta/js/routes.js, it looks like strings marked with ```visible: false``` are not added to the various string arrays.  Perhaps we can double check with a dev version in a  test with the translation utility?

**Comment by jessegreenberg** [comment]
I think that I misunderstood the behavior of ```visible: false```, it looks like this flag is more for cuing information to the translator and the string will still be visible in the translation utility.

To make sure that acccessibility strings are not in the translatable set, they should be removed from the json file, and hard coded in the sim.  This could be done *in the release branch* so that accessibility strings are still in the .json file in master.

**Comment by jessegreenberg** [comment]
I should mention that we don't want accessibility strings to be translatable because the content is still in prototype phase.  Keyboard navigation for the motion screens is incomplete, and the design for accessibility in the Net Force screens should be updated based on what we have learned from recent testing and discussions.

**Comment by jessegreenberg** [comment]
Changing title of issue to reflect this.

**Comment by jessegreenberg** [comment]
This will need to be done on the release branch before a test task can be prepared.

**Comment by jessegreenberg** [comment]
Done for the 2.0 branch in the commit above, closing.
